packages/react/src/feature-app-container.tsx

```diff
@@ -50,6 +50,12 @@ export interface FeatureAppContainerProps {
    * defined.
    */
   readonly idSpecifier?: string;
+
+  /**
+   * A Feature App config object that is intended for the specific Feature App
```

Comment:
This sound to me like a "Feature App config object" is something like a type. But in reality this could be anything...